interview-questions.md

```diff
@@ -16,44 +16,44 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: A git is the data and what it is contained in. Github is a place or cloud for the data to be stored making it easier to be shared and managed.
 
-Researched answer:
+Researched answer: Git is a free open-sourced version control system widely used in the programming world. It allows people to contribute to the same code. It was created to track changes in code. GitHub is the hosting platform for the Git. 
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: Booleans are true or false statements. Logical operators && (and) and || (or) will return a boolean value. 
 
-Researched answer:
+Researched answer: Equality operators compare one value to the other and judge if the values equate to the same thing. If they do equate to the same a Boolean value of true is returned if not we get a Boolean value of false.
 
-1. What is the difference between an array's index and its value?
+3. What is the difference between an array's index and its value?
 
-Your answer:
+Your answer: an arrays index starts the count at zero and includes spaces. The value is a count of the characters itself.
 
-Researched answer:
+Researched answer: The value in an array is an elemement. The content of the element is the value. The index is the location of the value. The index of each value is like an address for the value.
 
 4. Compare and contrast arrays and strings. Please include ways in which they are the same and ways in which they are different.
 
-Your answer:
+Your answer: While both arrays and strings are peices of data, strings are more of a statement and arrays are more of a vessel in which to store data.
 
-Researched answer:
+Researched answer: In JavaScript, arrays and strings have a lot of similar properties. They both are a collection of items, both have a length property, both are zero-indexed. But strings and arrays have many differences. It is often convenient to convert string into arrays and vice-versa.
```

Comment:
This question was asking you to compare and contrast, so make sure to include what differences there are in your answer.